src/js/modules/infragistics.ui.editors.js

```diff
@@ -7871,11 +7894,39 @@
 		_handleSpinDownEvent: function () { // DateEditor
 			this.spinDown(1);
 		},
+		_serializeDate: function (sDate) {
+			if (this.options.dataMode === "date") {
+				if (this.options.enableUTCDates) {
```

Comment:
It does exactly that. The JavaScript [`Date.prototype.toISOString()`](http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.5.43) version always outputs the UTC time zone value (supposedly to simplify implementations), which is why we also use a standard polyfill (`editor._toLocalISOString()` as seen below) to produce a version with the local zone.